docs/elements.md

```diff
@@ -275,6 +275,47 @@ Attribute | Type | Default | Description
 
 -----
 
+## `pl-dropdown` element
+
+Select the correct answer from a drop-down **select** menu list of potential answers. The potential options are listed in the inner HTML of a <pl-answer></pl-answer> element (ie. <pl-answer>Possible Answer 1</pl-answer>).
+
+#### Sample Element
+
+![](elements/pl-dropdown.png)
+
+**question.html**
+
+```html
+An <pl-dropdown sort="random" correct-answer="socrates">
+  <pl-answer correct="true">unexamined</pl-answer>
+  <pl-answer>examined</pl-answer>
+  <pl-answer>insatiable</pl-answer>
+</pl-dropdown> life is not worth living. <p></p>
+```
+
+**server.py**
+```python
+def generate(data):
+
+    # Correct Answers
+    data['correct_answers']['socrates'] = 'unexamined'
+```
+
+#### Customizations
+
+Attribute | Type | Default | Description
+--- | --- | --- | ---
+`correct-answer` | string | - | The key of the correct answer.
+`weight` | integer | 1 | Weight to use when computing a weighted average score over elements.
+`sort` | string | no sorting | Options are 'random', 'ascend', and 'descend' for drop-down answers.
+
+#### Example implementation
+
+- [demoOverlayDropdown](https://github.com/PrairieLearn/PrairieLearn/tree/master/exampleCourse/questions/demoOverlayDropdown)
+- [elementDropdown](https://github.com/PrairieLearn/PrairieLearn/tree/master/exampleCourse/questions/elementDropdown)
```

Comment:
Could you use reference links here? 

In particular, change:

```suggestion
- [demoOverlayDropdown]
- [elementDropdown]
```

At the bottom of the file, please add:

```md
[elementDropdown]: https://github.com/PrairieLearn/PrairieLearn/tree/master/exampleCourse/questions/elementDropdown
```

See:

https://github.com/PrairieLearn/PrairieLearn/pull/2494/files#diff-51a6a2f59d6deec90c5c3b10d067c8eeL1506-L1507

(Recent PR for `pl-overlay`)

Reply:
Will do!

Reply:
Done in ef10eeee81234fcc70f5055d4cff6d586afe9773.